ansq/tcp/reader.py

```diff
@@ -16,7 +16,7 @@ def __init__(
         topic: str,
         channel: str,
         nsqd_tcp_addresses: Sequence[str],
-        connection_options: Mapping[str, Any] = None,
+        connection_options: ConnectionOptions = ConnectionOptions(),
```

Comment:
```suggestion
        connection_options: Optional[ConnectionOptions] = None,
```

Passing mutable objects as defaults might lead to unexpected behavior, classic example: 
```python
def foo(bar=[]):
    ...
```

Reply:
Agree. I've implemented the structure as frozen first. I'll look into weither there will be a possibility to make it frozen after https://github.com/lig/ansq/pull/1 is merged.

Reply:
Addressed in https://github.com/lig/ansq/commit/a2d462e68e3294aecbd70094603a303ae46f2e0d

Reply:
Looks good 👍🏻